packages/Dropdown/package.json

```diff
@@ -28,6 +28,10 @@
     "classnames": "^2.3.2",
     "react-aria": "^3.21.0",
     "react-spring": "^9.5.5",
+    "react-popper": "^2.3.0",
+    "@types/react-dom": "17.0.13"
```

Comment:
this type should be in the devDependencies

Reply:
I sent something on slack to you. it complained when I did this and said it shouldn't be in devDependencies:
![image](https://user-images.githubusercontent.com/110498164/208138899-5a973cac-956b-4895-87c6-247c2d641463.png)